tools/generator/cmd/v2/common/fileProcessor.go

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package common
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/tools/generator/autorest/model"
+	"github.com/Azure/azure-sdk-for-go/tools/generator/common"
+	"github.com/Azure/azure-sdk-for-go/tools/internal/exports"
+	"github.com/Masterminds/semver"
+)
+
+const (
+	sdk_generated_file_prefix         = "zz_generated_"
+	autorest_md_swagger_url_prefix    = "- https://github.com/Azure/azure-rest-api-specs/blob/"
+	autorest_md_module_version_prefix = "module-version: "
+	swagger_md_module_name_prefix     = "module-name: "
+)
+
+var (
+	v2BeginRegex                   = regexp.MustCompile("^```\\s*yaml\\s*\\$\\(go\\)\\s*&&\\s*\\$\\(track2\\)")
```

Comment:
I'm not entirely if this is everywhere in readmes already but if possible we should change this to v2 as well instead of track2.

Reply:
For now, all readme in swagger use 'track2' to indicate the new sdk version config and also some other scripts use 'track2' to recognize this config. So I just reserve the 'track2' in regex expression and also add 'v2' string for future gradually convertion.